ConanInstall.cmake

```diff
@@ -64,30 +64,32 @@ if(NOT CONAN_OPENSTUDIO_ALREADY_RUN)
   conan_cmake_run(REQUIRES
     ${CONAN_READLINE}
     ${CONAN_QT}
-    OpenSSL/1.1.0g@conan/stable
+    openssl/1.1.0l # ruby 2.5.5 won't work with 1.1.1d, so use 1.1.0l here to try to force every package to align on the same as ruby
     # Track NREL/stable in general, on a feature branch this could be temporarily switched to NREL/testing
-    openstudio_ruby/2.5.5@nrel/stable
-    boost_asio/${BOOST_VERSION}@bincrafters/stable
-    boost_program_options/${BOOST_VERSION}@bincrafters/stable
-    boost_regex/${BOOST_VERSION}@bincrafters/stable
-    boost_filesystem/${BOOST_VERSION}@bincrafters/stable
-    boost_crc/${BOOST_VERSION}@bincrafters/stable
-    boost_algorithm/${BOOST_VERSION}@bincrafters/stable
-    boost_uuid/${BOOST_VERSION}@bincrafters/stable
-    boost_log/${BOOST_VERSION}@bincrafters/stable
-    boost_numeric_ublas/${BOOST_VERSION}@bincrafters/stable
-    boost_functional/${BOOST_VERSION}@bincrafters/stable
-    boost_geometry/${BOOST_VERSION}@bincrafters/stable
-    pugixml/1.9@bincrafters/stable
-    jsoncpp/1.8.4@theirix/stable
-    zlib/1.2.11@conan/stable
-    fmt/5.2.1@bincrafters/stable
-    sqlite3/3.27.2@bincrafters/stable
-    cpprestsdk/2.10.13@bincrafters/stable
-    websocketpp/0.8.1@jmarrec/stable # TODO: Temp to avoid hitting https://github.com/bincrafters/community/issues/1069
+    openstudio_ruby/2.5.5@jmarrec/testing    # TODO: Temp #@nrel/stable
```

Comment:
Revamped this one as well, including making upstream changes to conan-ruby_installer, patching ruby source itself, etc.

https://github.com/NREL/conan-openstudio-ruby/pull/20